src/test/java/de/tum/in/www1/artemis/metis/linkpreview/LinkPreviewIntegrationTest.java

```diff
@@ -24,8 +24,7 @@ class LinkPreviewIntegrationTest extends AbstractSpringIntegrationIndependentTes
     // this link will return null for all fields because it does not include OG tags
     private static final String GOOGLE_URL = "https://google.com";
 
```

Comment:
this still calls a foreign url

Reply:
all of them do, I tried it with artemis URLs but then the test failed, I hope someone fixes this when someone takes on #8054 as I don't want to loose much time right now